pandas/core/generic.py

```diff
@@ -3008,7 +3008,7 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
         See Also
         --------
         read_csv : Load a CSV file into a DataFrame.
-        to_excel : Load an Excel file into a DataFrame.
+        read_excel : Load an Excel file into a DataFrame.
```

Comment:
I don't think this will render a link as `read_excel` is not defined in the same module. 

In any case probably easier to keep `to_excel` and update the description accordingly

Reply:
Description updated.